core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -164,13 +168,20 @@ coreos:
             After=decrypt-assets.service
             {{- end}}
             {{if .Etcd.DisasterRecovery.SupportsEtcdVersion .Etcd.Version -}}
-            {{/* can be `Wants` if you like etcd-member to not stop when etcdadm-reconfigure failed */}}
-            BindsTo=etcdadm-reconfigure.service etcdadm-update-status.service
+            # record failing state
+            OnFailure=etcdadm-check.service
```

Comment:
Yes, this makes sense because then a failed etcd member can be marked as so earlier. More concretely, `etcdadm-check` gets triggered immediately after `etcd-member` failed, without waiting for the next interval of `etcdadm-check.timer`.